apps/coscrad-frontend/src/components/dynamicViews/dynamicResourceDetailView/DataPresenters/defaultPresenter.tsx

```diff
@@ -0,0 +1 @@
+export const defaultPresenter = (data: unknown) => <div>{JSON.stringify(data)}</div>;
```

Comment:
Could we use `replace(/\"/g, "")` to remove quotes?

Reply:
That's a good suggestion. I'd probably make the regex specific to opening and closing quotes. Maybe what's better is to use String(...) instead?

Reply:
On second thought, let's go with your solution for now. If there's ever an edge case, we can tweak it from there.